```
lib/private/Files/View.php:380 PhanUndeclaredClassMethod Call to method __construct from undeclared class \OC\Files\Storage\Storage\local
lib/private/Files/View.php:381 PhanUndeclaredClassMethod Call to method readdir from undeclared class \OC\Files\Storage\Storage\local
settings/Application.php:99 PhanUndeclaredClassMethod Call to method __construct from undeclared class \OC\Settings\View
Makefile:215: recipe for target 'test-php-phan' failed
```

--- a/.drone.yml
+++ b/.drone.yml
@@ -114,15 +114,6 @@ pipeline:
       matrix:
         TEST_SUITE: lint
 
-  php-cs-fixer:
-    image: owncloudci/php:${PHP_VERSION}
-    pull: true
-    commands:
-      - make test-php-style
-    when:
-      matrix:
-        TEST_SUITE: php-cs-fixer
-
   php-phan:
     image: owncloudci/php:7.1
     pull: true

--- a/lib/private/Files/View.php
+++ b/lib/private/Files/View.php
@@ -48,6 +48,7 @@ namespace OC\Files;
 use Icewind\Streams\CallbackWrapper;
 use OC\Files\Mount\MoveableMount;
 use OC\Files\Storage\Storage;
+use OC\Files\Storage\Local;
 use OC\User\RemoteUser;
 use OCP\Constants;
 use OCP\Events\EventEmitterTrait;
@@ -377,7 +378,7 @@ class View {
 	 * @return mixed
 	 */
 	public function readdir($handle) {
-		$fsLocal = new Storage\Local(['datadir' => '/']);
+		$fsLocal = new Local(['datadir' => '/']);
 		return $fsLocal->readdir($handle);
 	}
 

--- a/settings/Application.php
+++ b/settings/Application.php
@@ -49,6 +49,7 @@ use OC\Settings\Middleware\SubadminMiddleware;
 use OCP\AppFramework\App;
 use OCP\IContainer;
 use OCP\Util;
+use OC\Files\View;
 
 /**
  * @package OC\Settings

